modules/administration/pages/content-lifecycle.adoc

```diff
@@ -220,6 +220,39 @@ For more information on how to set up a live patching project, see xref:administ
 
 
 
+=== AppStream modules with defaults
+
+When you want to have all the modules available in a modular repository included in your project, you can automatically add them using this filter template.
+
+When applied, this template creates an AppStream filter per module and its default stream.
+
+If this process is done from the project's page, the filters are added to the project automatically. Otherwise, the created filters can be listed in menu:Content Lifecycle[Filters] and be added to any project as needed.
+
+Each individual filter can be edited to select a different module stream, or removed altogether to exclude that module from the target repositories.
+
+[IMPORTANT]
+====
+Since not all module streams are compatible with each other, changing individual streams may prevent successful resolution of modular dependencies. When this happens, the filters pane in the project details page will show an error describing the problem, and the build button will be disabled until all the module selections are compatible.
```

Comment:
```suggestion
Because not all module streams are compatible with each other, changing individual streams may prevent successful resolution of modular dependencies. When this happens, the filters pane in the project details page will show an error describing the problem, and the build button will be disabled until all the module selections are compatible.
```